section/algorithms.html

```diff
@@ -435,4 +435,21 @@ <h4>Generate HTTP Signature</h4>
             </pre>
         </aside>
     </div>
+
+    <div class="algorithm">
+        To <dfn>set as structured header</dfn> (|headerName|, |key|, |value:sequence of bytes|) for a |request:Request|,
+        <ol>
+            <li>Let |headerValue:structured field dictionary| be a <a data-cite="RFC9651#dictionary">structured field dictionary</a> with a single key |key| with corresponding value |value|.</li>
+            <li>Let |header| be a [=tuple=] of |headerName| and |headerValue|.</li>
+            <li>[=header list/Set a structured field value=] |header| to the [=request/header list=] of the |request|. The |headerValue| must be <a data-cite="RFC9651#name-serializing-a-dictionary">serialized</a> as described in [[RFC9651]].</li>
```

Comment:
RFC9651 reference should probably go into note section, because [=header list/Set a structured field value=] specifies that for a structure field value, right?